### PR TITLE
Bids 2060/sticky header table scroll

### PR DESCRIPTION
**Pull request by MauserBitfly**
### <samp>🤖 Generated by Copilot at 8d8425f</samp>

No summary available (Limit exceeded: required to process 58486 tokens, but only 50000 are allowed per call)

### CODE REVIEW

**Review by LuccaBitfly — CHANGES_REQUESTED** [review]
Nice changes, but I still have some issues regarding on the "implement mobile view & new styles" side of this PR:

After talking to @fly-d-dog, we should make sure that all headings look as similar as possible, in a radical way. They should look like this:
image
More specifically: Main header in 1.5 rem Montserrat, sub header in 1.125rem Montserrat, normal font color, ads above the heading, spacing like in the figma (don't know if I can post the link here) etc..

This applies to EVERY page you touched. Unique headings on pages like on `/stakingServices` or `/burn` should also be replaced, pages with no headings like `/tools/broadcast` should also get one.

I have commented on some issues I noticed below, but these may not include all discrepancies. Please make sure that all headings look the same.

There are also some pages you haven't touched, e.g. `block.html` still has breadcrumbs. Is this part of this BIDS?

File: templates/validators.html
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/validators.html:1]: It works for me, but I also would not want to fix an error here that is already fixed in the main branch, as the fix should be merged into the newfrontend branch anyway at some point in the future. (preferable after this branch is merged into the netfrontend branch)

File: templates/validators_leaderboard.html [hunk 1]
```diff
@@ -81,19 +81,11 @@
...
-            <ol class="breadcrumb font-size-1 mb-0" style="padding:0; background-color:transparent;">
-              <li class="breadcrumb-item"><a href="/" title="Home">Home</a></li>
-              <li class="breadcrumb-item"><a href="/validators" title="Validators">Validators</a></li>
-              <li class="breadcrumb-item active" aria-current="page">Leaderboard</li>
-            </ol>
-          </nav>
+          <h4 class="h4 mb-1 mb-md-0"><i class="fas fa-medal"></i> Validator Staking Leaderboard</h4>
         </div>
-        The Validator Leaderboard is ordered by the 7 day income by default.
+        <div class="sub-page-title">The Validator Leaderboard is ordered by the 7 day income by default.</div>
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/validators_leaderboard.html]: fixed

File: templates/validatorRewards.html [hunk 2]
```diff
@@ -28,22 +28,22 @@
...
+    <div class="d-md-flex justify-content-md-between">
+      <div class="page-title">
+        <h4 class="h4 mb-1 mb-md-0"><i class="far fa-money-bill-alt mr-2"></i>Validator Income History</h4>
       </div>
-      <p class="mb-4">Get the income history of your validators or subscribe to our monthly income report</p>
+
+      <nav aria-label="breadcrumb">
+        <ol class="breadcrumb font-size-1 mb-0" style="padding:0; background-color:transparent;">
+          <li class="breadcrumb-item"><a href="/" title="Home">Home</a></li>
+          <li class="breadcrumb-item active" aria-current="page">Income History</li>
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/validatorRewards.html]: removed

File: templates/slot/slotfinder.html [hunk 3]
```diff
@@ -7,13 +7,11 @@
...
-    <div class="my-3">
-      <div class="d-md-flex py-2 justify-content-md-between">
-        <h1 class="h4 mb-1 mb-md-0">
-          <span class="ml-1 mr-1"><i class="fas fa-cube mr-2"></i>Slot Finder<span style="color:grey; font-size: 70%; padding-left: 10px;" id="live-highest-block"></span></span>
-        </h1>
-      </div>
+  <div class="container">
+    <div class="page-title">
+      <h4>
+        <span class="mr-1"><i class="fas fa-cube mr-2"></i>Slot Finder<span style="color:grey; font-size: 70%; padding-left: 10px;" id="live-highest-block"></span></span>
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/slot/slotfinder.html]: fixed

File: templates/relays.html [hunk 4]
```diff
@@ -81,11 +83,11 @@ <h1>Relays:</h1>
             </div>
           </div>
         </div>
-        <div class="row mt-4">
+        <div class="row mb-d">
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/relays.html:87]: fixed the padding above the blocks headline.
The big space between the text and the next table is due to the tabbar on the right side

File: templates/relays.html [hunk 5]
```diff
@@ -16,10 +16,12 @@
...
   <section>
     <div class="container">
-      <div class="h-100 py-4">
-        <div class="row mt-4">
+      <div class="h-100">
+        <div class="row">
           <div class="col-md-12">
-            <h1>Relays:</h1>
+            <div class="page-title">
+              <h4 class="h4">Relays:</h4>
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/relays.html]: fixed

File: templates/pools_rocketpool.html [hunk 6]
```diff
@@ -458,25 +458,14 @@
...
-            </ol>
-          </nav>
-        </div>
-      </div>
-      <h1 class="mt-2 mb-5 text-nowrap" style="font-size: 1.8rem; letter-spacing: 2px;">
-        <a href="https://rocketpool.net/"
+      <div class="page-title">
+        <a href="https://rocketpool.net/" class="h4"
           >Rocket Pool
           <i class="fas fa-rocket"></i>
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/pools_rocketpool.html]: ok

File: templates/deposits.html [hunk 7]
```diff
@@ -140,21 +140,14 @@
 {{ define "content" }}
   {{ with .Data }}
     <div class="container my-2">
-      <div class="d-md-flex py-2 justify-content-md-between mb-3">
+      <div class="d-md-flex page-title justify-content-md-between">
         <div class="heading">
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/deposits.html]: fixed

File: templates/broadcaststatus.html [hunk 8]
```diff
@@ -20,13 +20,17 @@
...
+      <div id="r-banner" info="{{ $.Meta.Templates }}"></div>
+      <section>
         <div class="row">
           <div class="col-md-12">
             <div class="row justify-content-center">
-              <div class="col-md-12 col-lg-8 text-center">
-                <h4 class="h4 pb-3">{{ .JobTitle }}<br />{{ .Job.ID }}</h4>
+              <div class="col-md-12 col-lg-8">
+                <div class="page-title">
+                  <h4 class="h4">{{ .JobTitle }}</h4>
```
Comment (LuccaBitfly) [inline templates/broadcaststatus.html:31]: Spacing is different to other headings, should probably look like `/settings`

File: templates/user/webhooks.html [hunk 9]
```diff
@@ -58,35 +59,29 @@
...
+                    fill: none;
+                  }
+                </style>
+              </defs>
+              <path d="M24,26a3,3,0,1,0-2.8164-4H13v1a5,5,0,1,1-5-5V16a7,7,0,1,0,6.9287,8h6.2549A2.9914,2.9914,0,0,0,24,26Z" />
+              <path d="M24,16a7.024,7.024,0,0,0-2.57.4873l-3.1656-5.5395a3.0469,3.0469,0,1,0-1.7326.9985l4.1189,7.2085.8686-.4976a5.0006,5.0006,0,1,1-1.851,6.8418L17.937,26.501A7.0005,7.0005,0,1,0,24,16Z" />
+              <path d="M8.532,20.0537a3.03,3.03,0,1,0,1.7326.9985C11.74,18.47,13.86,14.7607,13.89,14.708l.4976-.8682-.8677-.497a5,5,0,1,1,6.812-1.8438l1.7315,1.002a7.0008,7.0008,0,1,0-10.3462,2.0356c-.457.7427-1.1021,1.8716-2.0737,3.5728Z" />
+              <rect class="webhook-cls-1" data-name="&lt;Transparent Rectangle&gt;" id="_Transparent_Rectangle_" />
+            </svg>
+            Webhooks
```
Comment (LuccaBitfly) [inline templates/user/webhooks.html:79]: Spacing differs from other headings

**Review by LuccaBitfly — CHANGES_REQUESTED** [review]
Overall I like :)
There are still some minor issues left though 😅

File: templates/correlations.html [hunk 10]
```diff
@@ -164,20 +164,12 @@
...
-            <li class="breadcrumb-item">Statistics</li>
-            <li class="breadcrumb-item active" aria-current="page">Correlations</li>
-          </ol>
-        </nav>
-      </div>
+  <div class="container">
+    <div id="r-banner" info="{{ $.Meta.Templates }}"></div>
+    <div class="page-title">
+      <h1 class="h4">
+        <span class="ml-1 mr-1"><i class="fas fa-chart-line mr-2"></i>Correlations</span>
```
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/correlations.html]: removed left spacing

File: templates/pools/pools.html
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply templates/pools/pools.html:1]: added a header

File: templates/broadcaststatus.html (hunk 8 above)
Comment (MauserBitfly) [reply templates/broadcaststatus.html]: moved the header now to the left

File: templates/user/webhooks.html (hunk 9 above)
Comment (MauserBitfly) [reply templates/user/webhooks.html:79]: tried to make it more compliant ... problem is the big button on the right that

**Comment by MauserBitfly** [comment]
> Overall I like :) There are still some minor issues left though 😅

Thanks made some changed accordingly

**Review by LuccaBitfly — APPROVED** [review]
lgtm

**Pull request merged**